modules/ROOT/pages/errors/gql-errors/index.adoc

```diff
@@ -155,6 +155,10 @@ Status description:: error: data exception - invalid value type
 
 Status description:: error: data exception - invalid date, time, or datetime function field name
 
+=== xref:errors/gql-errors/22G0I.adoc[22G01]
```

Comment:
This file is autogenerated. I'll run the script before I merge it. Don't need to worry about it.